diagnostic.md

```diff
@@ -9,17 +9,25 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+String: let variable = "string"
+number: let variable = 2
+null: let variable = ""
+undefined: let variable
+
+
 ## Question 2
 
 ```js
 let c = 5
 let d = 2
 c = c + d
-
 ```
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d` and explain what they do.
 
+c = 7
+the expression on the right is evaluated before the "=" assignement
+
```

Comment:
The two operators are `=` and `+`

`=` assigns a value
`+` combines two values